src/paas_charm/springboot/charm.py

```diff
@@ -287,6 +303,13 @@ def _create_app(self) -> App:
             A new App instance.
         """
         charm_state = self._create_charm_state()
+        if charm_state.integrations.saml:
```

Comment:
Could we merge a few if statements here to reduce nested code?
```suggestion
if charm_state.integrations.saml and charm_state.integrations.saml.signing_certificate:
    ...
```

Reply:
@alithethird I'm guessing this will be implementd next PR?